docs/cli-help.txt

```diff
@@ -22,14 +22,11 @@ optional arguments:
   -v, --verbose         This flag can be used multiple times to incrementally
                         increase the verbosity of output, e.g. -vvv. (default:
                         -2)
-  -t, --text            Use the text output instead of the curses UI.
```

Comment:
We often change the documentation for reasons other than a new release. We could delay publishing changes until a release has been done, but blocking all improvements to our documentation to keep this file in lockstep seems undesirable to me.